docs/README.md

```diff
@@ -2541,7 +2528,42 @@ npm init
 npm install --save @boostercloud/framework-provider-aws-infrastructure
 ```
 
-The implementation of `InfrastructureRocket` might vary from one provider to the other, but in AWS it only requires two functions:
+The schema of an *Infrastructure Rocket* project should look something like this:
+
+```text
+rocket-your-rocket-name-aws-infrastructure
+├── package.json
+├── src
+    ├── lambdas
+    ├── index.ts
+    └── your-main-class.ts
+
+```
+
+In `<your-main-class>.ts` is where you define the two functions that AWS requires:
+
+```typescript
+export class YourMainClass {
+  public static mountStack(params: YourRocketParams, stack: Stack, config: BoosterConfig): void {
+    /* CDK code to expand your Booster infrastructure */
+  }
+  public static unmountStack?(params: YourRocketParams, utils: RocketUtils): void {
+    /* Optional code that runs before removing the stack */
+  }
+}
+```
+
+Let's look in more detail these two special functions:
+
+- **mountStack**: This function will run when you deploy your Booster application. Here you can use the [CDK](https://docs.aws.amazon.com/cdk/latest/guide/home.html) code you need to extend the Booster functionality as you like. As we can see, it receives three params:
+  - `params`: The parameters required by your *Infrastructure Rocket* initializator, you will receive them from your Booster app's `config.ts` file.
+  - `stack`: An initialized AWS CDK stack that you can use to add new resources. Check out [the Stack API in the official CDK documentation](https://docs.aws.amazon.com/cdk/latest/guide/stacks.html#stack_api). This is the same stack instance that Booster uses to deploy its resources, so your resources will automatically be deployed along with the Booster's ones on the same stack. 
+  - `config`: It includes properties of the Booster project that is about to be deployed.
+
+
+- **unmountStack**: It will run when you run the `boost nuke` command. When you nuke your Booster application, all the resources added by your rocket are automatically destroyed along with the application stack, but there are some situations on which it's convenient to delete or move the contents of the resources created by your Rocket. In the `unmountStack` function you'll have the opportunity to run any code before deleting the stack. This function receives an utils object with the same tools that Booster uses to perform common actions like emptying the contents of an S3 bucket (Non-empty buckets are kept by default when a stack is deleted).
```

Comment:
````
- **unmountStack**: It will run when you run the `boost nuke` command. When you nuke your Booster application, all the resources added by your rocket are automatically destroyed along with the application stack, but there are some situations where you might or need to specify any additional step in the deletion process. The `unmountStack` function will run the code you intend for this purpose. For instance, in AWS, before destroying your stack (where you have some S3 buckets) you need to first empty them in order to delete them. You can accomplish this action in the `unmountStack` method.

````